### PR TITLE
Update of Rate Sheet XML for Year 62 forward

### PR DESCRIPTION
**Pull request by jonesgaohsu**
#### Rationale
Update of Rate Sheets Labels

#### Related Pull Requests
* 

#### Changes
Update of All NIH Public Rate Sheets

### CODE REVIEW

File: onprc_billingpublic/resources/queries/onprc_billing_public/NIHIndustryRates.query.xml [hunk 1]
```diff
@@ -13,38 +13,39 @@
...
                     </column>
                     <column columnName="year5">
-                        <columnTitle>YR66: 5/1/25 to 4/30/26</columnTitle>
+                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
                         <formatString>$#,##0.00</formatString>
                     </column>
 
                     <column columnName="year6">
-                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
+                        <columnTitle>YR68: 5/1/26=7 to 4/30/28</columnTitle>
```
Comment (labkey-jeckels) [inline onprc_billingpublic/resources/queries/onprc_billing_public/NIHIndustryRates.query.xml]: Typo here
```suggestion
                        <columnTitle>YR68: 5/1/27 to 4/30/28</columnTitle>
```

File: onprc_billingpublic/resources/queries/onprc_billing_public/NIHRateSheet.query.xml [hunk 2]
```diff
@@ -19,37 +19,37 @@
...
                     </column>
                     <column columnName="year5">
-                        <columnTitle>YR66: 5/1/25 to 4/30/26</columnTitle>
+                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
                         <formatString>$#,##0.00</formatString>
                     </column>
 
                     <column columnName="year6">
-                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
+                        <columnTitle>YR68: 5/1/26=7 to 4/30/28</columnTitle>
```
Comment (labkey-jeckels) [inline onprc_billingpublic/resources/queries/onprc_billing_public/NIHRateSheet.query.xml]: Same typo here.
```suggestion
                        <columnTitle>YR68: 5/1/27 to 4/30/28</columnTitle>
```

File: onprc_billingpublic/resources/queries/onprc_billing_public/NIHRateConfig.query.xml [hunk 3]
```diff
@@ -13,43 +13,43 @@
...
                     </column>
                     <column columnName="year5">
-                        <columnTitle>YR66: 5/1/25 to 4/30/26</columnTitle>
+                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
                         <formatString>$#,##0.00</formatString>
                     </column>
 
                     <column columnName="year6">
-                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
+                        <columnTitle>YR68: 5/1/26=7 to 4/30/28</columnTitle>
```
Comment (labkey-jeckels) [inline onprc_billingpublic/resources/queries/onprc_billing_public/NIHRateConfig.query.xml]:
```suggestion
                        <columnTitle>YR68: 5/1/27 to 4/30/28</columnTitle>
```

File: onprc_billingpublic/resources/queries/onprc_billing_public/NIHRates_ReducedFA.query.xml [hunk 4]
```diff
@@ -13,43 +13,43 @@
...
                     </column>
                     <column columnName="year5">
-                        <columnTitle>YR66: 5/1/25 to 4/30/26</columnTitle>
+                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
                         <formatString>$#,##0.00</formatString>
                     </column>
 
                     <column columnName="year6">
-                        <columnTitle>YR67: 5/1/26 to 4/30/27</columnTitle>
+                        <columnTitle>YR68: 5/1/26=7 to 4/30/28</columnTitle>
```
Comment (labkey-jeckels) [inline onprc_billingpublic/resources/queries/onprc_billing_public/NIHRates_ReducedFA.query.xml]:
```suggestion
                        <columnTitle>YR68: 5/1/27 to 4/30/28</columnTitle>
```

**Review by labkey-jeckels — DISMISSED** [review]

**Comment by jonesgaohsu** [comment]
Code cleaned up and ready for review

**Comment by labkey-jeckels** [comment]
@jonesgaohsu you fixed the typo and pushed in onprc_billingpublic/resources/queries/onprc_billing_public/NIHRateSheet.query.xml but the other files haven't been corrected yet

**Comment by jonesgaohsu** [comment]
Josh is this ready to be merged or am I missing a step

**Comment by labkey-ankurj** [comment]
Hi @jonesgaohsu, Josh is out this week, seems like there are suggestions to these files as well from Josh, that are pending corrections- 
onprc_billingpublic/resources/queries/onprc_billing_public/NIHIndustryRates.query.xml
onprc_billingpublic/resources/queries/onprc_billing_public/NIHRateConfig.query.xml
onprc_billingpublic/resources/queries/onprc_billing_public/NIHRates_ReducedFA.query.xml

**Review by jonesgaohsu — COMMENTED** [review]
I believe I have gotten all of them thought I had made these earlier

**Review by labkey-ankurj — APPROVED** [review]

**Comment by jonesgaohsu** [comment]
Ability to squash and merge this item appears to be locked

**Comment by labkey-keith** [comment]
Hi Gary, I think that you should be able to squash and merge now.

**Comment by jonesgaohsu** [comment]
Item was squashed and merged into default

**Pull request merged**